step00-start/src/main/java/com/google/example/resizecodelab/view/MainActivity.kt

```diff
@@ -90,13 +90,13 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun getSuggestedProducts() : Array<Suggestion> {
-        return arrayOf(Suggestion(getString(R.string.label_product_name2), R.drawable.gregarious),
-                Suggestion(getString(R.string.label_product_name3), R.drawable.byzantium),
-                Suggestion(getString(R.string.label_product_name4), R.drawable.cratankerous),
-                Suggestion(getString(R.string.label_product_name5), R.drawable.sunsari),
-                Suggestion(getString(R.string.label_product_name6), R.drawable.squiggle),
-                Suggestion(getString(R.string.label_product_name7), R.drawable.tenacious),
-                Suggestion(getString(R.string.label_product_name8), R.drawable.venemial))
+        return arrayOf(Suggestion("Gregarious Grogglestock", R.drawable.gregarious),
```

Comment:
Why not string resources?

Reply:
This is the first step in simplifying the project, in the next step you'll see why I've killed the strings. All sample data will be provided by a single class instead of half and half like the current pattern.